wildlifelicensing/static/wl/js/wl.bootstrap-treeview.js

```diff
@@ -0,0 +1,1237 @@
+define(['jQuery'], function ($) {
```

Comment:
You should probably put a comment at the beginning telling that is a copy of bootstrap-treeview (with url to original code) and explaining what you changed.
Also I think you should probably keep the licence comments.